x/sequencer/types/messages.go

```diff
@@ -5,6 +5,8 @@ import (
 	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/decred/dcrd/dcrec/edwards"
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
```

Comment:
what are these repos? why not using the tendermint ones?

Reply:
afaik tendermint does not expose any validation function. what i did in this pr is to parse the keys to those libraries that will validate the public keys